assessments/projects/custodian/joint-review.md

```diff
@@ -0,0 +1,900 @@
+      * [Expected Attacker Capabilities](#expected-attacker-capabilities)
+      * [Attack Risks and Effects](#attack-risks-and-effects)
+      * [Example Attack Scenarios](#example-attack-scenarios)
+      * [Security Degradation](#security-degradation)
+      * [Compensating Mechanisms](#compensating-mechanisms)
+      * [STRIDE Evaluation](#stride-evaluation)
+         * [STRIDE Classification Considered](#stride-classification-considered)
+         * [Control Recommendation (SEVERITY: Medium)](#control-recommendation-severity-medium)
+         * [Control Recommendation (SEVERITY: Medium)](#control-recommendation-severity-medium-1)
+   * [Threat Model](#threat-model)
+* [Secure Development Practices](#secure-development-practices)
+   * [Security Issue Resolution](#security-issue-resolution)
+      * [Currently Open Security Issues and Vulnerabilities](#currently-open-security-issues-and-vulnerabilities)
+   * [Hands-on Review](#hands-on-review)
+   * [Roadmap](#roadmap)
+      * [Recommended Project Next Steps](#recommended-project-next-steps)
+      * [Recommended CNCF Requests](#recommended-cncf-requests)
+   * [Appendix](#appendix)
+      * [Case Studies](#case-studies)
+
+
+## Metadata
+
+| Project Home                    | https://github.com/cloud-custodian                                                                               |
+|---------------------------------|------------------------------------------------------------------------------------------------------------------|
+| Security Provider               | Yes, however many users implement Cloud Custodian for non security purposes, i.e., cost optimization, governance |
+| Languages                       | Python, YAML |
+| Incubation PR                   | https://github.com/cncf/toc/pull/480 |
+| TAG-Security Assessment Request | https://github.com/cncf/tag-security/issues/307 |
+| Self-Assessment Document        | https://docs.google.com/document/d/1s88ifDtFJzGX1O_ve6HM6_vbTkpsWDKWRDuxt9RHdcs/ |
+| SBOM                            | [poetry configuration in pyproject.toml](https://github.com/cloud-custodian/cloud-custodian/blob/master/pyproject.toml), [poetry lock file](https://github.com/cloud-custodian/cloud-custodian/blob/master/poetry.lock), [pip requirements file](https://github.com/cloud-custodian/cloud-custodian/blob/master/requirements.txt), [release process](https://cloudcustodian.io/docs/developer/packaging.html) |
+
+## Reviewers and Participants
+
+* Security Review Lead:
+* Security Co-Lead:
+* Security Co-Lead:
+* Security Reviewer:
+* Security Reviewer:
+* Project Lead:
+* Project Member:
+* Project Member:
+
+
+### Security links
+
+| Item | Ref or URL |
+| -- | -- |
+| Security Mailing List  | security@cloudcustodian.io |
+| Threat Model (2019) | https://docs.google.com/document/d/1S9zQZaT6G1TA3IAx6YNL0f7G938xaFZ-bziszhuxMZg |
+
+## Overview
+
+Cloud Custodian is a YAML DSL-based stateless rules engine for cloud auditing,
```

Comment:
Please expand DSL here then abbreviate in parens.

Reply:
fixed